### ISSUE TITLE
get_label_range_var(args) returns new RangeVar pointers

### ISSUE THREAD

**Issue by susano0** [apache/age#337]
The name of the function: 
```
RangeVar *get_label_range_var(char *graph_name, Oid graph_oid, char *label_name)
```
 in `ag_label.c` is misleading. 

This function generates a new RangeVar pointer rather than returning the one that was first generated when a label was created, I believe its name should be changed to something like "`create_label_range_var`" or "`generate_label_range_var`".

Also, the function `get_label_range_var` should be fixed to return the correct pointer.

**Comment by jrgemignani** [comment]
Good point. I will point out that the description for the function does point out that it creates one. But, the name of the function should also be more clear.

Additionally, another name worth considering could be `make_label_range_var` which is in line with a lot of PG functions.
